Sources/MockoloFramework/Models/IfMacroModel.swift

```diff
@@ -15,35 +15,88 @@
 //
 
 final class IfMacroModel: Model {
-    let name: String
+    struct Clause {
+        let condition: String?  // nil == else clause
+        let entities: [(String, Model)]
+        let clauseType: ClauseType
+
+        enum ClauseType: Hashable {
+            case `if`
+            case elseif(order: Int)
+            case `else`
+            
+            init?(_ clauseType: String) {
+                if clauseType == "if" {
+                    self = .if
+                } else if clauseType.hasPrefix("elseif-"), let order = Int(String(clauseType.dropFirst(7))) {
+                    self = .elseif(order: order)
+                } else if clauseType == "else" {
+                    self = .else
+                } else {
+                    return nil
+                }
+            }
+            
+            /// order in if-elseif-else block
+            ///
+            /// `999_999` corresponds to `else` clause
+            var order: Int {
+                switch self {
+                case .if:
+                    0
+                case .elseif(let order):
+                    order
+                case .else:
+                    999_999
+                }
+            }
```

Comment:
`order` is necessary to support multiple elseIf condition.

```swift
#if FOO
import Foo
#elseif BAR
import Bar
#elseif BAZ
import Baz
#endif
```

In the example above, I added an order property to record whether BAR precedes BAZ.